core/config/project_settings.cpp

```diff
@@ -145,16 +145,14 @@ String ProjectSettings::localize_path(const String &p_path) const {
 		return p_path.simplify_path();
 	}
 
-	DirAccess *dir = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
+	DirAccessRef dir = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
 
```

Comment:
In theory I guess we should, though the assumption is probably that aside from API that runs very early (like `ProjectSettings::setup()`), it should be a given that `DirAccess::create()` should work. It's more `DirAccess::open()` which is risky as it can easily fail if the path doesn't exist (which is used to check for path existence).